### PR TITLE
editing docs in preparation for 2.5.2 release candidate 1

### PR DESCRIPTION
**Pull request by GodloveD**
**Description of the Pull Request (PR):**

Title says it all.

- ✅ I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- ✅I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- ✅I have tested this PR locally with a `make test`
- ✅This PR is NOT against the project's master branch
- ✅I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- ✅This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

**Review by jmstover — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,21 +12,31 @@ and changes prior to that are (unfortunately) done retrospectively. Critical ite
...
+ - fix a bug with passwd abort #1580
+ - allow user to override singularity.conf "mount home = no" with --home option
+   #1496
+ - Improve debugging output #1535
+ - Fix some bugs in bind mounting #1525
+ - Define PR_(S|G)ET_NO_NEW_PRIVS in user space so that these features will 
+   work with kernels that implement them (like Cray systems) #1506
+ - Create /dev/fd and standard streams symlinks in /dev when using minimal dev
+   mount or when specifying -c/-C/--contain option #1420
+ - fixed * expansion during app runscript creation #1486
```
Comment (DrDaveD) [inline CHANGELOG.md:31]: Isn't #1212 also included?  add comment to bind path option about being ignored with --contain

**Comment by GodloveD** [comment]
@dctrud Yes it is.  There are several others as well.  This changelog is not an exhaustive list.  Most people who submitted PRs did not edit the changelog and I had to go back and add the changes by hand.  I chose to leave out things that were documentation.  But I can add that one in if you think it is important.

**Comment by DrDaveD** [comment]
I never edit the CHANGELOG because I never know what release something is going to go into.

It's OK with me if you want to leave something out, I just thought you had forgotten.

**Comment by GodloveD** [comment]
Since this is time-sensitive I'm going to merge.  Thanks @jmstover and @DrDaveD.

**Pull request merged**